server/server.go

```diff
@@ -70,31 +70,12 @@ func NewServer(ctx *Context) (*Server, error) {
 	if ctx == nil {
 		return nil, util.Error("ctx must not be null")
 	}
-	// Determine hostname in case it hasn't been specified in -rpc or -http.
+	// Determine hostname in case it hasn't been specified in -http.
 	host, err := os.Hostname()
```

Comment:
Switched back to the old behavior of adding in hostname if no host or ip address was specified.